### PR TITLE
Wiring AnimationDesigner with Animation module

### PR DESCRIPTION
**Pull request by anand-sundaram**
Not sure if this code works (need to create a save button somewhere to test it). 

I have just expanded the `AnimationDesignerDelegate` protocol and implemented it in `GridController`, based on what I understand of @ngzhian's code :)

### CODE REVIEW

**Comment by anand-sundaram** [comment]
Ok. It doesn't seem to break anything, and it fixes a few things:
- Gets the animation type segments to read from the enum in Animation Module
- Allows for the display of fresh grids for different frames of the timeline
- Prevents animation and sound from playing on a pad which already has animation and sample assigned to it, when clicked in the designer mode.
- Provides a way to keep track of and save the animation (though this bit has not been tested)

Please review and merge!

File: MIDIchlorians/MIDIchlorians/GridController.swift [hunk 1]
```diff
@@ -41,6 +41,9 @@ class GridController: NSObject {
 
     internal var gridCollectionView: GridCollectionView
     internal var colour: Colour?
+    internal var animationSequence: AnimationSequence
+    internal var animationName: String = "Default Animation"
```
Comment (ngzhian) [inline MIDIchlorians/MIDIchlorians/GridController.swift]: magic constant

**Review by ngzhian — APPROVED** [review]
lgtm!

**Comment by ngzhian** [comment]
Oh and to resolve the conflict, keep master, but change that one line to:
`if let colour = colours[selectedFrame][pad]`

**Pull request merged**